muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/communicator/CommunicatorJsPluginMessages_en.properties

```diff
@@ -39,6 +39,7 @@ plugin.communicator.label.create.error.alreadyexists = A label with that name al
 plugin.communicator.label.create.error.empty = Name cannot be empty
 plugin.communicator.label.remove.question = Do you want to remove label: "{0}"?
 plugin.communicator.label.edit.caption = Edit label
+plugin.communicator.label.editLabelDialog.name = Name
```

Comment:
plugin.communicator.label.editLabelDialog.name

Wouldn't this be better like this? :

plugin.communicator.dialog.editLabel.name

Reply:
Can be changed, however there are multiple locale strings currently that are used inside of dialog but are not specified with locale path. We have issue for unifying every locale string we have and I would like to leave this change as it is and take care of it when fixing every locale string.